requirements/adjust_versions.py

```diff
@@ -3,16 +3,18 @@
 import sys
 from typing import Dict, Optional
 
+# IMPORTANT: this list needs to be sorted in reverse
 VERSIONS = [
     dict(torch="1.10.0", torchvision="0.11.*", torchtext=""),  # nightly
+    dict(torch="1.9.1", torchvision="0.10.0", torchtext="0.10.0"),
     dict(torch="1.9.0", torchvision="0.10.0", torchtext="0.10.0"),
+    dict(torch="1.8.2", torchvision="0.9.1", torchtext="0.9.1"),
     dict(torch="1.8.1", torchvision="0.9.1", torchtext="0.9.1"),
     dict(torch="1.8.0", torchvision="0.9.0", torchtext="0.9.0"),
     dict(torch="1.7.1", torchvision="0.8.2", torchtext="0.8.1"),
     dict(torch="1.7.0", torchvision="0.8.1", torchtext="0.8.0"),
     dict(torch="1.6.0", torchvision="0.7.0", torchtext="0.7"),
 ]
-VERSIONS.sort(key=lambda v: v["torch"], reverse=True)
```

Comment:
This sort is not correct because it puts `1.10.0`at the end. It would need to be:

```python
from packaging.version import Version
VERSIONS.sort(key=lambda v: Version(v["torch"]), reverse=True)
```

However, the dependency is not installed at this point.

I think it's fine that we keep it sorted manually.